go.mod

```diff
@@ -28,6 +28,7 @@ require (
 	github.com/lithammer/shortuuid/v3 v3.0.4 // indirect
 	github.com/oklog/ulid v1.3.1 // indirect
 	github.com/pmezard/go-difflib v1.0.0 // indirect
+	github.com/stretchr/objx v0.1.1 // indirect
```

Comment:
Why do we have this new dependency? Is it test dependency or runtime one?

Reply:
This is testcase dependency only.